### ISSUE TITLE
Save action on build settings page should provide feedback to the user

### ISSUE THREAD

**Issue by jamesnetherton** [docker/hub-beta-feedback#184]
If I click the save button on the build settings page, it changes background color to a darker shade of blue and this remains until focus on the button is lost.

It's not obvious whether the save operation was actually successful or not. There should be a message or some feedback given to inform the user.

To add my 2 cents on the rest of the design, I really don't like it. Mostly due to the colors used. Bright, blue text on blue backgrounds just doesn't work.

**Comment by ChristopherBiscardi** [comment]
Agreed. Labelling this as a bug because we should have much better feedback for the user in this case.

--- 

As for the rest of the design: We are working on the "blue on blue" issue although I don't know when we'll ship a fix overall or what that fix will end up looking like.

**Comment by nhsiehgit** [comment]
immediate feedback for build-settings pages has been updated and fixed. 
changing label from `bug` to `needs improvement` since this is definitely something that can still be improved.

**Comment by lucadruda** [comment]
Functionalities mentioned in the issue are not valid anymore for the current version of Docker Hub.
Closing the issue.
Feel free to reopen it if this is still a problem.
